lib/pkgcloud/openstack/database/client/instances.js

```diff
@@ -320,3 +320,70 @@ exports.setVolumeSize = function setVolumeSize(instance, newSize, callback) {
     }), callback);
   });
 };
+
+// Enable the root user for the given database instance.
+// ### @instance {string | Object} The ID of the instance or a instance or Instance class (required)
+// ### @callback {Function} Function to continue, the call is cb(error, res)
+exports.enableRootUser = function(instance, callback) {
```

Comment:
Bleh. For context: http://docs.rackspace.com/cdb/api/v1.0/cdb-devguide/content/POST_createRoot__version___accountId__instances__instanceId__root_Database_Instances.html